robottelo/cli/subscription.py

```diff
@@ -69,7 +69,7 @@ def refresh_manifest(cls, options=None):
         return result
 
     @classmethod
-    def manifest_history(cls, options=None, expect_csv=True):
+    def manifest_history(cls, options=None):
```

Comment:
It is not used anywhere inside this function, because this I decided to drop it.